apps/main/src/lend/components/PageLoanManage/ManageLoanTabs.tsx

```diff
@@ -0,0 +1,103 @@
+import LoanBorrowMore, { LoanBorrowMoreWrapped } from '@/lend/components/PageLoanManage/LoanBorrowMore'
+import LoanCollateralAdd, { LoanAddCollateralTab } from '@/lend/components/PageLoanManage/LoanCollateralAdd'
+import LoanCollateralRemove, { LoanRemoveCollateralTab } from '@/lend/components/PageLoanManage/LoanCollateralRemove'
+import LoanRepay, {
+  LoanRepayFromCollateralTab,
+  LoanRepayFromWalletTab,
+} from '@/lend/components/PageLoanManage/LoanRepay'
+import LoanSelfLiquidation from '@/lend/components/PageLoanManage/LoanSelfLiquidation'
+import networks from '@/lend/networks'
+import { type MarketUrlParams, PageContentProps } from '@/lend/types/lend.types'
+import { useClosePositionTab } from '@/llamalend/features/manage-soft-liquidation/hooks/useClosePositionTab'
+import { useImproveHealthTab } from '@/llamalend/features/manage-soft-liquidation/hooks/useImproveHealthTab'
+import { ClosePosition } from '@/llamalend/features/manage-soft-liquidation/ui/tabs/ClosePosition'
+import { ImproveHealth } from '@/llamalend/features/manage-soft-liquidation/ui/tabs/ImproveHealth'
+import { useManageLoanMuiForm, useManageSoftLiquidation } from '@ui-kit/hooks/useFeatureFlags'
+import { t } from '@ui-kit/lib/i18n'
+import { type FormTab, FormTabs } from '@ui-kit/shared/ui/FormTabs/FormTabs'
+
+type ManageLoanProps = PageContentProps<MarketUrlParams>
+
+export const ImproveHealthTab = ({ rChainId, rOwmId }: ManageLoanProps) => (
```

Comment:
You're only using these directly down below in `LendManageSoftLiquidationMenu`, so I wouldn't mind if you just inlined them. Saves from us from coming up with a name and an unnecessary export